ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -589,10 +590,10 @@ extension PillButtonBar: UIPointerInteractionDelegate {
         guard let index = region.identifier as? Int else {
             return
         }
-        if let window = window, customPillButtonBackgroundColor == nil, index < buttons.count {
+        if window != nil, customPillButtonBackgroundColor == nil, index < buttons.count {
```

Comment:
Not sure we need this `window` check at all anymore. The old color APIs needed a window to fetch, but we shouldn't at this point.